src/send-request.js

```diff
@@ -70,8 +80,9 @@ export const xhr = ({ url, data, headers, options, captureMetrics, callback, ret
                     callback(response)
                 }
             } else {
-                const error = 'Bad HTTP status: ' + req.status + ' ' + req.statusText
-                console.error(error)
+                if (onXHRError instanceof Function) {
+                    onXHRError(req)
+                }
```

Comment:
Nit: I think for simple checks it's better to use `typeof` over `instanceof`, or you could even replace this with a oneliner like `onXHRError && onXHRError(req)`.

Reply:
It can't be `onXHRError && onXHRError(req)` (I _think_) because we're checking specifically that it is a function before we try to call it